### ISSUE TITLE
Is there any way to broadcast a client computer to the other clients?

### ISSUE THREAD

**Issue by toddobryan** [epoptes/epoptes#220]
I teach programming and it would be really nice to be able to have the class look at different students' answers to different problems.

Is there any way to broadcast a computer other than the one controlling Epoptes? If not currently, is such a thing possible? I would love to see this feature, as I could randomly share a students screen with other students and have them explain what they have.

**Comment by robark** [comment]
Yes. Simply broadcast to everyone except the student who you wish to share the screen. Or alternatively, broadcast to everyone and ask the particular student to close their broadcast window. Then simply  monitor or assist that student. Everyone will see that students screen.

**Comment by alkisg** [comment]
This is not a bug in epoptes, converting into a discussion.
